src/workspacer/Keybinds/KeybindManager.cs

```diff
@@ -56,9 +57,38 @@ public KeybindManager(IConfigContext context)
             thread.Start();
         }
 
+        public void ShowKeybindWarning(string warningMessage)
```

Comment:
This method doesn't show the dialog per current implementation, it *toggles* it. 

Is that intentional?

Reply:
I copied the structure and naming from `ShowKeybindDialog` and retained it for consistency. happy to adjust as it can be simplified to 
```csharp
 public void ShowKeybindWarning(string warningMessage)
        {
            if (_keybindWarning == null)
            {
                _keybindWarning = new TextBlockMessage("workspacer keybinds", "Warning, duplicate keybinds!", warningMessage, new List<Tuple<string, Action>>()
                {
                    new Tuple<string, Action>("ok", () => { }),
                });
          
                _keybindWarning.Show();
            }
        }
```

Reply:
That looks better, indeed.